posts/deployToEcsWithGithubAction.md

```diff
@@ -0,0 +1,177 @@
+---
+authors:
+- Amine Ouali Alami
+tags:
+- Spring Boot
+- JHipster
+- GitHub
+- GitHub Actions
+- DevOps
+- AWS
+- Amazon ECS
+- AWS Fargate
+- Amazon ECR
+
+date: 2020-20-10T20:16:10.000Z
+title: "Deploy to ECS using GitHub Actions"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/01/deployToEcsWithGithubAction.jpg
+---
+
+Building a DevOps platform nowadays can be challenging for Architect and Infrastructure teams, due to the integration complexity with many platforms: source control, ci/cd, artifact repositories, quality, SAST/SCA, deployment ...  , Many DevOps leader believes that the challenge of DevOps is to reduce the number of the platforms involved in one pipeline.
+That's why tech giants are trying to integrate all the pipeline steps into their platforms.
+In this Blog, we will deploy a simple spring boot application to ECS using the new GitHub feature called GitHub Actions.
+
+# What is GitHub Actions
+GitHub Actions enables you to create a custom software development lifecycle pipelines directly in your Github repository.
+you can write individual tasks, called actions, and combine them to create a custom pipeline.
+Behind the scene, your pipeline is run on GitHub-hosted machines, called runners.
+
+
+# Create your first pipeline with Github actions
+On your GitHub repository select the Actions tab.
+GitHub propose popular pipelines to start with, we will choose: Deploy to Amazon ECS
+
+![01](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/01/deployToEcsWithGithubAction-01.png)
+
+It will create in your source code the file /.github/workflows/aws.yml that represents your GitHub Actions.
+We added two more steps to set up the JDK 8 and the maven command to build the Spring Boot JAR file.
```

Comment:
JDK 8 is old now! It's for dinosaurs like us! :trollface:  If it's not too much work and impact, maybe you can update the example to use the latest?